modules/virt-about-tekton-tasks-operator.adoc

```diff
@@ -9,3 +9,33 @@
 The `tekton-tasks-operator` deploys example pipelines showing the usage of OpenShift Pipelines for VMs. It also deploys additional OpenShift Pipeline tasks that allow users to create VMs from templates, copy and modify templates, and create data volumes.
 
 image::cnv_components_tekton-tasks-operator.png[tekton-tasks-operator components]
+
+.tekton-tasks-operator components
+[cols="1,1"]
+|===
+|*Component* |*Description*
+
+|`deployment/create-vm-from-template`
+|	Creates a VM from an {VirtProductName} template.
+
+|`deployment/copy-template`
+|	Copies an {VirtProductName} template.
+
+|`deployment/modify-vm-template`
+|	Creates or removes an {VirtProductName} template.
+
+|`deployment/modify-data-object`
+|	Creates or removes DataVolumes or DataSources.
+
+|`deployment/cleanup-vm`
+|	Executes a script or a command on a VM, then stops or deletes the VM once the script or command is complete.
```

Comment:
Might want to replace "execute" with "run" 

According to the ISG:
Use with caution
execute   verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.